### PR TITLE
Make readme content more generic

### PR DESCRIPTION
**Pull request by ruanxin**

### CODE REVIEW

File: docs/creating-test-environment.md [hunk 1]
```diff
@@ -15,9 +15,10 @@ Make sure to have two `KUBECONFIG` compliant client configurations at hand, one
 
 Provision two compliant kyma Clusters with the `kyma-cli`:
 
+`[gardener account kubeconfig]`: the access credential which can be downloaded from [gardener account](https://dashboard.garden.canary.k8s.ondemand.com/account). 
 ```sh
-kyma provision gardener gcp --name op-kcp --project jellyfish -s gcp-jellyfish-secret -c .kube/kubeconfig-garden-jellyfish.yaml
-kyma provision gardener gcp --name op-skr --project jellyfish -s gcp-jellyfish-secret -c .kube/kubeconfig-garden-jellyfish.yaml
+kyma provision gardener gcp --name op-kcp --project [gardener project] -s [secret for gcp] -c [gardener account kubeconfig]
```
Comment (jakobmoellersap) [inline docs/creating-test-environment.md:20]: why not directly use bash variables here?

Comment (ruanxin) [inline docs/creating-test-environment.md:20]:
```suggestion
kyma provision gardener gcp --name op-kcp --project ${gardener_project} -s ${gcp_secret} -c ${gardener_account_kubeconfig}
```
you mean like this?

File: docs/creating-test-environment.md [hunk 2]
```diff
@@ -15,9 +15,10 @@ Make sure to have two `KUBECONFIG` compliant client configurations at hand, one
...
+kyma provision gardener gcp --name op-skr --project [gardener project] -s [secret for gcp] -c [gardener account kubeconfig]
```
Comment (ruanxin) [inline docs/creating-test-environment.md:21]:
```suggestion
kyma provision gardener gcp --name op-skr --project ${gardener_project} -s ${gcp_secret} -c ${gardener_account_kubeconfig}
```

File: docs/creating-test-environment.md [hunk 3]
```diff
@@ -15,9 +15,10 @@ Make sure to have two `KUBECONFIG` compliant client configurations at hand, one
 
 Provision two compliant kyma Clusters with the `kyma-cli`:
 
+`[gardener account kubeconfig]`: the access credential which can be downloaded from [gardener account](https://dashboard.garden.canary.k8s.ondemand.com/account).
```
Comment (ruanxin) [inline docs/creating-test-environment.md:18]:
```suggestion
`${gardener_account_kubeconfig}`: the access credential which can be downloaded from [gardener account](https://dashboard.garden.canary.k8s.ondemand.com/account). 
```

**Review by jakobmoellersap — APPROVED** [review]

**Pull request merged**